.github/workflows/integration.yml

```diff
@@ -0,0 +1,54 @@
+name: Integration
+
+on:
+    push:
+        paths-ignore:
+            - '*.md'
+        branches:
+            - master
+            - '[0-9].[0-9]'
+    pull_request:
+        branches:
+            - master
+            - '[0-9].[0-9]'
+
+jobs:
+
+    build-and-test:
+        runs-on: ubuntu-latest
+        timeout-minutes: 60
+        strategy:
+            max-parallel: 15
+            fail-fast: false
+            matrix:
+                python-version: ['3.8', '3.9', '3,10', '3.11', '3.12']
```

Comment:
How should this list of versions correspond to the list in the pyproject.toml?  Should it cover the range specified in the pyproject.toml?

Reply:
Correct. Unfortunately there's nothing native to GitHub actions that allows you to specify a matrix via python troves. This is the next best approach.